scripts/user/add-user.ts

```diff
@@ -100,6 +100,7 @@ const checkParams = params => {
           'torosChambers',
           'urdasChambers',
           'vasquezsChambers',
+          'waysChambers',
```

Comment:
Note to self and future devs: this isn't necessary for adding the judge, which is done through Zendesk automation and python scripts that call the `/users` DAWSON endpoint to eventually hit the `createUserLambda`. This is for ensuring future added users can be validated appropriately via the new chamber section.

Reply:
Update: turns out we haven't been using the Zendesk automation for judges specifically; it has been done manually